cogs/shareLink.py

```diff
@@ -40,7 +40,7 @@ async def album(self, ctx: Context, *args: str):
     @classmethod
     def is_enabled(cls, configs: Dict[str, Any] = {}):
         return (
-            configs["ENABLE_SHARE_LINK"]
+            configs["ENABLE_SHARE_LINK"]        
```

Comment:
There is some whitespace generated here. Remove that. I will comment how to automatically format stuff